monitor.cpp

```diff
@@ -4,35 +4,66 @@
 #include <chrono>
 #include <iostream>
 using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
-
+/// @brief displayWarning function to display warning message
+void displayWarning() {
```

Comment:
Function naming: `displayWarning` sounds like it will display the warning message itself. But it's only grabbing the attention of the user, while the warning is displayed outside this function. Try giving a name that makes its purpose clear